### ISSUE TITLE
Unrounded Progress Container

### ISSUE THREAD

**Issue by withintheruins14** [iqnivek/react-circular-progressbar#48]
This looks great, I am about to drop it in but is it possible to style the progress element to have "hard edges"? The style of our site is boxy and nothing has border-radius soft looking styles. Thanks! I am looking for a style like this from react-progress-circle:
![screen shot 2018-03-02 at 9 28 37 am](https://user-images.githubusercontent.com/2576699/36906685-2ef4588a-1dfc-11e8-82d5-657878414f24.png)

I don't want to use the react-progress-circle component because it isn't as flexible in other areas. Thanks!

**Comment by withintheruins14** [comment]
The answer is to set the path subcomponent stroke-linecap property to 'square' from 'rounded'. Thanks! Great component.

**Comment by iqnivek** [comment]
Yup, that's the way to do it - glad you found the answer! I'm planning to add stuff like this to the README so it's a little easier to find in the future.
